pkg/controllers/gateway/gateway_controller.go

```diff
@@ -61,6 +63,7 @@ type HostService interface {
 	GetManagedZoneForHost(ctx context.Context, domain string, t traffic.Interface) (*v1alpha1.ManagedZone, string, error)
 	SetEndpoints(ctx context.Context, endpoints []gatewayv1beta1.GatewayAddress, dnsRecord *v1alpha1.DNSRecord, dnsPolicy *v1alpha1.DNSPolicy) error
 	CleanupDNSRecords(ctx context.Context, owner traffic.Interface) error
+	GetDNSRecordsFor(ctx context.Context, t traffic.Interface) ([]*v1alpha1.DNSRecord, error)
```

Comment:
I assume this change is no longer using this? I guess it's not a big deal to leave it, might be useful in the future. Depends if we want unused code left hanging about.

Reply:
oh yeah, I'll remove it